### PR TITLE
[backend] use component name in traces and fix warn issue.(#8352)(#8310)

### LINKED ISSUE #8310: Playbook "log data in standard output" warning option should use "warn" level instead of "warning"
## Description

Log level "warning" does not exist for the logger that we use in OpenCTI backend, so when the playbook component "Log data in standard output" is used with warning, the output is not logged, instead there is an error message.
```
[winston] Unknown logger level: warning
```

## Environment

1. OS (where OpenCTI server runs): { e.g. Mac OS 10, Windows 10, Ubuntu 16.4, etc. }
2. OpenCTI version: { e.g. OpenCTI 1.0.2 } 6.2.18
3. OpenCTI client: { e.g. frontend or python }
4. Other environment details:

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Create a playbook with a "Log data in standard output" component, at warning level
2. Trigger event to run playbook
3. Look at console output or logs.

## Expected Output

Truncated but this kind of output is expected with level:"warn".
```
{"bundle":{........},"category":"APP","level":"info","message":"[PLAYBOOK MANAGER] Logger component output","source":"backend","timestamp":"2024-09-11T13:38:29.898Z","version":"6.2.18"}
```

## Actual Output

```
[winston] Unknown logger level: warning
```

### LINKED ISSUE #8352: Playbook: Use component name in "Last execution traces"
## Use case

To debug the execution of a playbook, it would be preferable to use the “component name” specified by the end user in “Last execution traces” instead of using the “generic” component name. This is particularly important when you have a tree that uses the same component several times.

Capture d'écran 2024-09-16 082301

### PR DESCRIPTION
**Pull request by aHenryJard**
### Proposed changes

* Fix the 'warn' level
* Use component user define name in traces instead of generic component name.

### Related issues

* closes #8352
* closes #8310

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: opencti-platform/opencti-graphql/src/manager/playbookManager.ts [hunk 1]
```diff
@@ -156,8 +156,9 @@ export const playbookExecutor = async ({
       const end = utcDate();
       const durationDiff = end.diff(start);
       const duration = moment.duration(durationDiff);
+
```
Comment (aHenryJard) [inline opencti-platform/opencti-graphql/src/manager/playbookManager.ts]:
```suggestion
```

File: opencti-platform/opencti-graphql/src/modules/playbook/playbook-domain.ts [hunk 2]
```diff
@@ -178,6 +178,7 @@ export const playbookReplaceNode = async (context: AuthContext, user: AuthUser,
     }
     return n;
   });
+
```
Comment (aHenryJard) [inline opencti-platform/opencti-graphql/src/modules/playbook/playbook-domain.ts]:
```suggestion
```

File: opencti-platform/opencti-graphql/src/modules/playbook/playbook-components.ts [hunk 3]
```diff
@@ -110,7 +110,7 @@ const PLAYBOOK_LOGGER_COMPONENT_SCHEMA: JSONSchemaType<LoggerConfiguration> = {
       oneOf: [
         { const: 'debug', title: 'debug' },
         { const: 'info', title: 'info' },
-        { const: 'warning', title: 'warning' },
+        { const: 'warn', title: 'warn' },
```
Comment (marieflorescontact) [inline opencti-platform/opencti-graphql/src/modules/playbook/playbook-components.ts:113]: Seems ok but I don't know how to test this part?

File: opencti-platform/opencti-graphql/src/manager/playbookManager.ts [hunk 4]
```diff
@@ -157,7 +157,7 @@ export const playbookExecutor = async ({
       const durationDiff = end.diff(start);
       const duration = moment.duration(durationDiff);
       const observation: ObservationFn = {
-        message: `${nextStep.component.name.trim()} successfully executed in ${duration.humanize()}`,
+        message: `${nextStep.instance.name.trim()} successfully executed in ${duration.humanize()}`,
```
Comment (marieflorescontact) [inline opencti-platform/opencti-graphql/src/manager/playbookManager.ts:160]: locally tested ok 👌

File: opencti-platform/opencti-graphql/src/modules/playbook/playbook-components.ts (hunk 3 above)
Comment (aHenryJard) [reply opencti-platform/opencti-graphql/src/modules/playbook/playbook-components.ts:113]: you need a component 
image

if you test on master with 'warning', you will see an error in console log of opencti, when using this fix the bundle is in log with warn level.

Comment (marieflorescontact) [reply opencti-platform/opencti-graphql/src/modules/playbook/playbook-components.ts:113]: tested ok 👌

**Review by marieflorescontact — APPROVED** [review]

**Pull request merged**